File: model_renderer/lib/data/ModelViewType.dart
enum ModelViewType{
  Normal,
  Points,
  Mesh,
  X_RAY
}


class ModelViewTypeClass {
 static getStringFromViewType(ModelViewType type){
   switch(type) {
     case ModelViewType.Normal:
       return "Normal";
     case ModelViewType.Points:
       return "Points";
     case ModelViewType.Mesh:
       return "Mesh";
     case ModelViewType.X_RAY:
       return "X_RAY";
   }
 }
}

File: model_renderer/android/src/main/java/com/pranjal/model_renderer/ModelViewType.java
package com.pranjal.model_renderer;

enum ModelViewType{
    Normal,
    Points,
    Mesh,
    X_RAY
}


class ModelViewTypeClass{
    public static ModelViewType getViewTypeFromString(String a){
        switch (a){
            case "Normal":
                return ModelViewType.Normal;
            case "Points":
                return ModelViewType.Points;
            case "Mesh":
                return ModelViewType.Mesh;
            case "X_RAY":
                return ModelViewType.X_RAY;
        }
        return null;
    }
}
